akka-docs/src/test/java/jdocs/stream/IntegrationDocTest.java

```diff
@@ -405,6 +408,68 @@ public void actorRefWithAckExample() throws Exception {
     //#actorRefWithAck
   }
 
+  @Test
+  public void conflate() throws Exception {
+    //#conflate
+    TestPublisher.Probe<String> publisher = TestPublisher.probe(0, system);
+    TestSubscriber.ManualProbe<String> subscriber = TestSubscriber.manualProbe(system);
```

Comment:
I think those snippets should be small and to the point without showing testing stuff. I can see that in here you want to show how the result differ with demand. Perhaps then first have a very concise one-liner of the `conflate` and then the full example with some additional explanation.

Reply:
Ok makes sense, will do